samples/symbolication/src/symbolicationMain/kotlin/main.kt

```diff
@@ -21,6 +21,32 @@ fun archToCpuName(arch: String): cpu_type_t = when (arch) {
     else -> TODO("unsupported $arch")
 }
 
+fun sourceInfoString(owner: CValue<CSSymbolOwnerRef>, address: ULong): String? {
+    var sourceInfo: CValue<CSTypeRef>? = CSSymbolOwnerGetSourceInfoWithAddress(owner, address)
+    if (sourceInfo == null || CSIsNull(sourceInfo)) return null
```

Comment:
I guess `CSSymbolOwnerGetSourceInfoWithAddress` can't return `null`, so it makes sense to use `!!` here.